### PR TITLE
Release notes and version updates for 1.3.0

### PR DESCRIPTION
**Pull request by desagar**

### CODE REVIEW

File: content/en/docs/releasenotes/_index.md [hunk 1]
```diff
@@ -4,6 +4,40 @@ linkTitle: Release Notes
...
+Features:
+
+- Post-Install Updates: Configurations for DNS, certificate management, logging, ingress and Opensearch cluster configuration can be updated after a Verrazzano installation.
+- Added Jaeger Distributed Tracing.
+- Support for Kubernetes v1.22 and v1.23.
+- kube-prometheus-stack components are now part of Verrazzano and can be enabled if desired. These include Prometheus Operator, AlertManager, kube-state-metrics etc.
+
+Component version updates:
+
+- Cert-Manager v1.7.1
```
Comment (pfmackin) [inline content/en/docs/releasenotes/_index.md:17]: cert-manager

File: content/en/docs/releasenotes/_index.md [hunk 2]
```diff
@@ -4,6 +4,40 @@ linkTitle: Release Notes
...
+- Coherence Operator 3.2.5
+- Istio v1.13.2
+- Kiali v1.42.0
+- Nginx Ingress Controller v1.1.1
```
Comment (pfmackin) [inline content/en/docs/releasenotes/_index.md:21]: NGINX Ingress Controller

File: content/en/docs/releasenotes/_index.md [hunk 3]
```diff
@@ -4,6 +4,40 @@ linkTitle: Release Notes
...
+- Node Exporter v1.3.1
+- Prometheus v2.34.0
+- Rancher v2.6.4
+- WebLogic Kubernetes Operator v3.4.0
+
+Components Added:
+
+- AlertManager v0.24.0
+- Jaeger v1.32.0
+- Kube State Metrics v2.4.2
```
Comment (pfmackin) [inline content/en/docs/releasenotes/_index.md:31]: I think kube-state-metrics

File: content/en/docs/releasenotes/_index.md [hunk 4]
```diff
@@ -4,6 +4,40 @@ linkTitle: Release Notes
...
 ---
+### v1.3.0
+Features:
+
+- Post-Install Updates: Configurations for DNS, certificate management, logging, ingress and Opensearch cluster configuration can be updated after a Verrazzano installation.
+- Added Jaeger Distributed Tracing.
+- Support for Kubernetes v1.22 and v1.23.
+- kube-prometheus-stack components are now part of Verrazzano and can be enabled if desired. These include Prometheus Operator, AlertManager, kube-state-metrics etc.
+
+Component version updates:
```
Comment (kminder) [inline content/en/docs/releasenotes/_index.md:15]: Components Updated: 
To match Components Added:?
Rosemary has final say.

Comment (desagar) [reply content/en/docs/releasenotes/_index.md:15]: I used Component version updates because that's what was used before.

Comment (kminder) [reply content/en/docs/releasenotes/_index.md:15]: k

File: content/en/docs/releasenotes/_index.md [hunk 5]
```diff
@@ -4,6 +4,40 @@ linkTitle: Release Notes
 weight: 13
 draft: false
 ---
+### v1.3.0
+Features:
+
+- Post-Install Updates: Configurations for DNS, certificate management, logging, ingress and Opensearch cluster configuration can be updated after a Verrazzano installation.
+- Added Jaeger Distributed Tracing.
+- Support for Kubernetes v1.22 and v1.23.
+- kube-prometheus-stack components are now part of Verrazzano and can be enabled if desired. These include Prometheus Operator, AlertManager, kube-state-metrics etc.
```
Comment (kminder) [inline content/en/docs/releasenotes/_index.md:13]: I'm wondering how cert-manager and alert-manager should be referred to in the docs.  I can't find a conclusive trademark answer.

Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:13]: re-write on last part of the sentence -> and can be enabled; these include: Prometheus Operator, AlertManager, kube-state-metrics, and such.

File: content/en/docs/releasenotes/_index.md [hunk 6]
```diff
@@ -4,6 +4,40 @@ linkTitle: Release Notes
...
+- Coherence Operator 3.2.5
+- Istio v1.13.2
+- Kiali v1.42.0
+- NGINX Ingress Controller v1.1.1
+- Node Exporter v1.3.1
+- Prometheus v2.34.0
+- Rancher v2.6.4
+- WebLogic Kubernetes Operator v3.4.0
+
+Components Added:
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:27]: Components Added -> Components added

**Review by kminder — APPROVED** [review]
Approved.

File: content/en/docs/releasenotes/_index.md [hunk 7]
```diff
@@ -4,6 +4,40 @@ linkTitle: Release Notes
...
+- AlertManager v0.24.0
+- Jaeger v1.32.0
+- kube-state-metrics v2.4.2
+- Prometheus Adapter v0.9.1
+- Prometheus Operator v0.55.1
+- Prometheus Pushgateway v1.4.2
+
+Fixes:
+
+- Resolved an issue in the console UI with displaying multi-cluster applications when a managed cluster is partially registered.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:38]: multi-cluster -> multicluster

File: content/en/docs/releasenotes/_index.md [hunk 8]
```diff
@@ -4,6 +4,40 @@ linkTitle: Release Notes
...
+- Resolved an issue in the console UI with the display of the bobs-books sample WebLogic application.
```
Comment (rosemarymarano) [inline content/en/docs/releasenotes/_index.md:39]: bobs-books -> Bob's Books

**Review by rosemarymarano — COMMENTED** [review]
Edits and corrections.

**Pull request merged**